pkgs/top-level/perl-packages.nix

```
@@ -11014,8 +11014,10 @@ let
     };
     propagatedBuildInputs = [ commonsense ];
 
-    meta = with stdenv.lib; {
-      platforms = platforms.linux;
+    meta = {
+      description = "Scalable directory/file change notification for Perl on Linux";
+      license = with stdenv.lib.licenses; [ artistic1 gpl1Plus ];
```

```suggestion
      license = with stdenv.lib.licenses; [ artistic1 gpl1Plus ];
```

I'm unable to find any reference of a license for this module. Even metacpan says "License: unknown".

---

For MLEHMANN's modules he usually just says this under https://metacpan.org/source/MLEHMANN/Linux-Inotify2-2.2/COPYING

```
This module is licensed under the same terms as perl itself.
```

Similar with https://metacpan.org/release/AnyEvent ; I believe only Coro and EV have some exceptions.

---

Thanks for clarifying